### PR TITLE
chore, refactor, feat(egen): follows new template, simplifies code, adds cleanup step

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR includes the following updates to the `tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb` notebook:

- replace REGION with LOCATION
- add code to remove the resources (tensorboard, experiment, logs file)
- remove unnecessary code (for cleaning up the storage bucket)
- simplify the display-name variables
- follow the new template structure
- reorganize the sections according to the flow of the tutorial
- remove future tense
- simplify some sentences in the explanation for tensorboard instances
- move "what is Tensorboard" (informative section) to the beginning of the tutorial
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb [hunk 1]
```diff
@@ -161,226 +123,165 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "init_aip:mbsdk,all"
+        "id": "bea5ee30d9ca"
       },
       "source": [
-        "### Initialize the Vertex AI SDK for Python\n",
+        "## What is Vertex AI TensorBoard\n",
```
Comment (kittyabs) [inline notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb:255]: add question mark:
- What is Vertex AI TensorBoard?

File: notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb [hunk 2]
```diff
@@ -583,22 +484,13 @@
...
-      ]
-    },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "BkbB5GEI3Ge3"
       },
       "source": [
-        "### Create Vertex AI Tensorboard\n",
+        "## Create Vertex AI Tensorboard\n",
```
Comment (kittyabs) [inline notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb:493]: - change: Vertex AI Tensorboard
- to: Vertex AI TensorBoard

File: notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb [hunk 3]
```diff
@@ -685,23 +562,25 @@
...
+        "The left pane of the dashboard provides filtering capabilities that are active across all the views in the HParams dashboard. In this pane, you can:\n",
         "\n",
-        "- Filter which hyperparameters/metrics are shown in the dashboard\n",
-        "- Filter which hyperparameter/metrics values are shown in the dashboard\n",
-        "- Filter on run status (running, success, ...)\n",
-        "- Sort by hyperparameter/metric in the table view\n",
-        "- Number of session groups to show (useful for performance when there are many experiments)\n",
+        "- Filter which hyperparameters/metrics are shown in the dashboard.\n",
+        "- Filter which hyperparameter/metrics values are shown in the dashboard.\n",
+        "- Filter on run status (running, success etc.).\n",
```
Comment (kittyabs) [inline notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb:573]: add comma:
- (running, success, etc.)

File: notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb [hunk 4]
```diff
@@ -685,23 +562,25 @@
...
+        "* The **Parallel Coordinates View** shows each run as a line going through an axis for each hyperparemeter and metric. Click and drag the mouse on any axis to mark a region which highlights only the runs that pass through it. This can be useful for identifying which groups of hyperparameters are most important. The axes themselves can be re-ordered by dragging them.\n",
         "* The **Scatter Plot View** shows plots comparing each hyperparameter/metric with each metric. This can help identify correlations. Click and drag to select a region in a specific plot and highlight those sessions across the other plots.\n",
         "\n",
-        "A table row, a parallel coordinates line, and a scatter plot market can be clicked to see a plot of the metrics as a function of training steps for that session (although in this tutorial only one step is used for each run)."
+        "A table view, a parallel coordinates view, and a scatter plot matrix view are also provided. These views help to see the plots of the metrics as a function of training steps for that session (although in this tutorial only one step is used for each run)."
```
Comment (kittyabs) [inline notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb:583]: - change:  "A table view, a parallel coordinates view, and a scatter plot matrix view are also provided."
- to: "Also provided are a table view, a parallel coordinates view, and a scatter plot matrix view."

**Review by kittyabs — COMMENTED** [review]
several edits needed

File: notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb (hunk 4 above)
Comment (krishr2d2) [reply notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb:583]: This sentence seems unnecessary. The bullet point descriptions already cover these views. Hence, removing the sentence.

**Review by katiemn — APPROVED** [review]

**Pull request merged**